### PR TITLE
Modify sensors to return sets of TrueDetection types

### PR DESCRIPTION
**Pull request by oharrald-Dstl**
Allows for reference to the ground truth paths that produced the detections from the sensor.

Changes made in response to issue #196 

Note: Contains some format changes, import reordering in modified files.

### CODE REVIEW

**Review by jswright-dstl — APPROVED** [review]

File: stonesoup/sensor/radar/radar.py [hunk 1]
```diff
@@ -420,8 +429,8 @@ def rotate(self, timestamp):
 
         super().rotate(timestamp)
 
-        dwell_center_max = self.for_angle/2.0 - self.fov_angle/2.0
-        dwell_center_min = -self.for_angle/2.0 + self.fov_angle/2.0
+        dwell_center_max = self.for_angle / 2.0 - self.fov_angle / 2.0
+        dwell_center_min = -self.for_angle / 2.0 + self.fov_angle / 2.0
```
Comment (sdhiscocks) [inline stonesoup/sensor/radar/radar.py:433]: Format before was correct per PEP8

File: stonesoup/sensor/radar/tests/test_radar.py [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-    rotation_matrix = rotz(theta_z)@roty(theta_y)@rotx(theta_x)
+    rotation_matrix = rotz(theta_z) @ roty(theta_y) @ rotx(theta_x)
 
     xyz_rot = rotation_matrix @ xyz
     x = xyz_rot[0, 0]
     y = xyz_rot[1, 0]
     # z = 0  # xyz_rot[2, 0]
 
-    rho = np.sqrt(x**2 + y**2)
+    rho = np.sqrt(x ** 2 + y ** 2)
```
Comment (sdhiscocks) [inline stonesoup/sensor/radar/tests/test_radar.py:38]: Format before was right.

File: stonesoup/sensor/radar/radar.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 import copy
-from math import erfc
 from typing import Tuple, Set, Union
 
 import numpy as np
 import scipy.constants as const
+from math import erfc
```
Comment (sdhiscocks) [inline stonesoup/sensor/radar/radar.py:7]: This is standard library

Comment (oharrald-Dstl) [reply stonesoup/sensor/radar/radar.py:7]: The line `from typing import Tuple, Set, Union`?

Comment (sdhiscocks) [reply stonesoup/sensor/radar/radar.py:7]: `math` is standard library, so should be in first set of imports, where it was before.

**Review by sdhiscocks — APPROVED** [review]

**Pull request merged**